### PR TITLE
docs: historacle Avg spec

### PR DESCRIPTION
**Pull request by robert-zaremba**
## Description

Specification for storying AVG counters to be used in IBC Quota

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Review by gsk967 — APPROVED** [review]
LGTM

**Comment by gsk967** [comment]
@robert-zaremba  can you fix the readme lint issues

**Comment by robert-zaremba** [comment]
lint fixed

File: docs/design_docs/011-historacle-pricing.md [hunk 1]
```diff
@@ -38,26 +39,56 @@ Historacle pricing will provide an API for the leverage module to tell when pric
...
 
 - `Historic Stamp Period`: will determine how often exchange rates are stamped & stored, until the `Maximum Price Stamps` is met.
 - `Median Stamp Period`: will determine how often the Median and the Standard Deviations around the Median are calculated, which will also be stored in the state machine.
 
-### Maximums
+Hardcoded parameters:
 
-We define two Maximum values, which correspond to the most we will store of a measurement at a given time. This can be multiplied by their respective Epochs to find which length of time information is kept.
+- `AvgPeriod`: will determine the length of the window where we average prices.
+- `AvgShift`: will determine the time difference between average
```
Comment (rbajollari) [inline docs/design_docs/011-historacle-pricing.md:53]:
```suggestion
- `AvgShift`: will determine the time difference between averages
```

**Review by rbajollari — APPROVED** [review]
LGTM

**Pull request merged**